pkgs/applications/editors/bluej-bin/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, makeWrapper, unzip, jetbrains
+}:
+let
+  version = "414";
+in
+stdenv.mkDerivation {
+  name = "bluej-bin-${version}";
+  src = fetchurl {
```

Comment:
The jar file can be fetched directly
http://www.bluej.org/download/files/BlueJ-generic-420.jar
Note it's also possible to build from source, though I'm not sure whether/how the `nixpkgs.buildMaven` functions.

Reply:
BlueJ uses ant for building itself which makes it pretty difficult to deal with dependencies.

Reply:
> The jar file can be fetched directly
> http://www.bluej.org/download/files/BlueJ-generic-420.jar
The jar file is not the same as that from the deb package. It's a graphical installer.

Reply:
What steps does Debian take to avoid an interactive installer? Is it just a simple flag, or more involved?

Reply:
I have no idea. It's just another binary inside this package afaik; I found no makefile or something else that delivers this debian package. It's also not from the debian repositories, but from their website.

Reply:
Again... this makes sense, thank you for providing clarification.

We have a bad habit of not building java applications ourselves in nixpkgs (which would be nice to fix some day...) so what you've done is fine.